main.js

```diff
@@ -542,32 +603,295 @@ function ScrollToTop() {
+                    <ListItemText disableTypography primary="Usage" className={`pl-12 ${activeSubsections.findIndex((element) => element === "typography--usage") !== -1 ? classesMain.navitemselected : classesMain.navitem}`} />
+                    </ListItem>
+                </div>
+                }
+            </React.Fragment>
+                <React.Fragment key="navlink-fragment-5">
+                  <RouterLink to="/component" onClick={() => {
+                    setActiveMenuState("component");
+                    reduxStore.dispatch(clearNavAction());
+                  }}
+                  >
+                    <ListItem button key="navlink-5">
+                      <ListItemText disableTypography primary="Component" className={`pl-4 ${activeMenuState === "component" ? classesMain.navitemselected : classesMain.navitem}`} />
+                    </ListItem>
+                  </RouterLink>
+                  {activeMenuState === "component" &&
+                    <RouterLink key="navlink-sub-5-1" to="/colorclass" onClick={() => {
+                      reduxStore.dispatch(clearNavAction())
+                      reduxStore.dispatch(addNavAction("color-class"))
+                    }}>
+                      <ListItem button>
+                        <ListItemText disableTypography primary="Color Classes" className={`pl-12 ${activeSubsections.findIndex((element) => element === "color-class") !== -1 ? classesMain.navitemselected : classesMain.navitem}`} />
+                      </ListItem>
+                    </RouterLink>
+                  }
+                  {activeMenuState === "component" &&
+                    <RouterLink key="navlink-sub-5-2" to="/button" onClick={() => {
+                      reduxStore.dispatch(clearNavAction())
+                      reduxStore.dispatch(addNavAction("button"))
+                    }}>
+                      <ListItem button>
+                        <ListItemText disableTypography primary="Buttons" className={`pl-12 ${activeSubsections.findIndex((element) => element === "button") !== -1 ? classesMain.navitemselected : classesMain.navitem}`} />
+                      </ListItem>
+                    </RouterLink>
+                  }
+                  {activeMenuState === "component" &&
+                    <RouterLink key="navlink-sub-5-3" to="/card" onClick={() => {
+                      reduxStore.dispatch(clearNavAction())
+                      reduxStore.dispatch(addNavAction("card"))
+                    }}>
+                      <ListItem button>
+                        <ListItemText disableTypography primary="Card" className={`pl-12 ${activeSubsections.findIndex((element) => element === "card") !== -1 ? classesMain.navitemselected : classesMain.navitem}`} />
+                      </ListItem>
+                    </RouterLink>
+                  }
+                </React.Fragment>
+                {/* <React.Fragment key="navlink-fragment-6">
```

Comment:
Can probably just delete this instead of having it commented in here.

Reply:
done